docs/user-guide/configure-overview.md

```diff
@@ -1,14 +1,12 @@
-## Configuring Styx environment
-
-Styx supports a YAML-based configuration. A yaml file can be specified using a system property `YAML_CONFIG_LOCATION=your/file/path`.
-By default a file is taken from the classpath at `classpath:conf/environment/styx-config.yaml`.
-Regardless of file location, these properties can all be overwritten using system properties by the same name.
+## Configuring Styx
 
+Styx supports a YAML file-based configuration. The YAML file can be specified using the system property `YAML_CONFIG_LOCATION=your/file/path`.
+By default, the file is read from the classpath at `classpath:conf/environment/styx-config.yaml`.
+Additionally, all these properties can be overwritten using environment variables with the same name as the property.
```

Comment:
This is badly outdated information. The preferred (and only?) way is to pass the configuration file as a command line argument to the startup script. Please update accordingly.

Reply:
Actually, specifying the configuration file with "CONFIG_FILE" (as the quickstart explains) also work. I hadn't noticed it wasn't the same property. 

Should I remove the reference to "CONFIG_FILE" from both documents?

Reply:
As far as I can tell `YAML_CONFIG_LOCATION` is not used by Styx at all, so it should not appear in the documentation.